assets/scss/base/_table.scss

```diff
@@ -104,6 +104,11 @@ table {
     font-weight: 400;
   }
 }
+
+.table--heavy-header {
+    border-bottom: em(2) solid $govuk-blue-colour;
+}
```

Comment:
Is the default th styling not in effect the table__header? I would have thought that this is a variant of the default th? .table__header--heavy?